client/src/components/ResponsiveCarousel.tsx

```diff
@@ -0,0 +1,157 @@
+import * as React from 'react';
```

Comment:
tip for next time: wait until after review approval to change file names
that way reviewer can see changes you've made within the file